modal/volume.py

```diff
@@ -116,6 +116,58 @@ class VolumeInfo:
 class _VolumeManager:
     """Namespace with methods for managing named Volume objects."""
 
+    @staticmethod
+    async def create(
+        name: str,
```

Comment:
This is missing comments to document the parameters:

```suggestion
        name: str,  # Name to use for the new Volume
```